### PR TITLE
fix(package.json): update versions

### PR DESCRIPTION
**Pull request by CarolineGard**

### CODE REVIEW

File: packages/expandable/index.js [hunk 1]
```diff
@@ -89,9 +89,9 @@ class WarpExpandable extends kebabCaseAttributes(LitElement) {
               })}
               @click=${() => (this.expanded = !this.expanded)}
             >
-              <div class="${ccExpandable.title}">
+              <div class="flex justify-between align-center">
```
Comment (AnnaRybkina) [inline packages/expandable/index.js:92]: Woopsie

File: packages/expandable/index.js [hunk 2]
```diff
@@ -89,9 +89,9 @@ class WarpExpandable extends kebabCaseAttributes(LitElement) {
...
                 ${this.title
-                  ? html`<span class="${ccExpandable.titleType}">${this.title}</span>`
+                  ? html`<span class="h4">${this.title}</span>`
```
Comment (AnnaRybkina) [inline packages/expandable/index.js:94]: Same woopsie

File: package.json [hunk 3]
```diff
@@ -37,47 +37,47 @@
...
+    "@babel/eslint-parser": "7.22.5",
     "@chbphone55/classnames": "2.0.0",
     "@eik/cli": "^2.0.22",
-    "@semantic-release/changelog": "6.0.2",
+    "@semantic-release/changelog": "6.0.3",
     "@semantic-release/git": "10.0.1",
     "@warp-ds/uno": "^1.0.0-alpha.45",
-    "@warp-ds/component-classes": "1.0.0-alpha.106",
-    "autoprefixer": "10.4.13",
+    "@warp-ds/component-classes": "1.0.0-alpha.109",
```
Comment (AnnaRybkina) [inline package.json:47]: It is `1.0.0-alpha.110` already 😅

File: packages/expandable/index.js (hunk 1 above)
Comment (CarolineGard) [reply packages/expandable/index.js:92]: what!! good catch 😅

File: package.json [hunk 4]
```diff
@@ -37,47 +37,47 @@
...
+    "@semantic-release/changelog": "6.0.3",
     "@semantic-release/git": "10.0.1",
     "@warp-ds/uno": "^1.0.0-alpha.45",
-    "@warp-ds/component-classes": "1.0.0-alpha.106",
-    "autoprefixer": "10.4.13",
+    "@warp-ds/component-classes": "1.0.0-alpha.110",
+    "autoprefixer": "10.4.14",
     "cors": "2.8.5",
-    "cssnano": "5.1.14",
+    "cssnano": "6.0.1",
```
Comment (AnnaRybkina) [inline package.json:50]: Major upgrade here, have we done some smoke tests or something? Are we good with it?

Comment (CarolineGard) [reply package.json:50]: Actually, this one is not used anywhere in the project. I removed it 🙌

**Review by AnnaRybkina — APPROVED** [review]

**Pull request merged**